### ISSUE TITLE
How to extract only features of the TimeSformer given a video. Thanks in advance..

### ISSUE THREAD

**Issue by Wajahat0** [facebookresearch/TimeSformer#31]

**Comment by LemurPwned** [comment]
I saw couple potentially useful classes here `https://github.com/facebookresearch/TimeSformer/blob/main/timesformer/models/features.py`, they seem to be detached though -- the model itself doesn't have the `feature_info` property (aka `FeatureInfo` object). 
I wonder though if the actual intention of those classes was to transform models into feature extractors?

**Comment by gberta** [comment]
Sorry for the delay to respond. You could modify the tools/test_net.py and timesformer/models/vit.py function to do that. Basically, in timesformer/models/vit.py you would need to add a second output argument for VisionTransformer's forward method so that it would return features in addition to the logits. Then, in tools/test_net.py you could modify the code to save those returned features. Hope this helps!
